wherobots/db/driver.py

```diff
@@ -51,6 +50,7 @@ def connect(
     results_format: Union[ResultsFormat, None] = None,
     data_compression: Union[DataCompression, None] = None,
     geometry_representation: Union[GeometryRepresentation, None] = None,
+    ws_url: str = None,
```

Comment:
Why is this required, instead of calling `connect_direct()` directly?

The reason this parameter wasn't included in `connect()` is that it creates ambiguity between the rest of the parameters (like runtime/region) and the runtime you'd actually connect to when providing a `ws_url`, which may not match those choices.